Directory.Build.props

```
@@ -14,8 +14,8 @@
     <PackageIcon>icon.png</PackageIcon>
     <IncludeSymbols>true</IncludeSymbols>
     <RepositoryType>git</RepositoryType>
-    <VersionPrefix>0.3.1</VersionPrefix>
-    <VersionSuffix></VersionSuffix>
+    <VersionPrefix>0.3.2</VersionPrefix>
+    <VersionSuffix>preview-241031</VersionSuffix>
```

```suggestion
    <VersionSuffix></VersionSuffix>
```

I would maybe keep these preview strings internal unless we publish them on nuget? There are other ways to generate prerelease strings automatically which we have developed for the main bonsai repo which we can discuss incorporating here if necessary.